### PR TITLE
GoDepTest: Update the expected version of golang.org/x/sys

### PR DESCRIPTION
**Pull request by mnonnenmacher**

### CODE REVIEW

File: analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ project:
     dependencies:
     - id: "GoDep::github.com/mattn/go-isatty:v0.0.12"
       linkage: "STATIC"
-    - id: "GoDep::golang.org/x/sys:05986578812163b26672dabd9b425240ae2bb0ad"
+    - id: "GoDep::golang.org/x/sys:8d3cce7afc34617998104db7c4b58c2de9e77215"
```
Comment (sschuberth) [inline analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml:23]: Just curious, why's this needed now that all Go tests are synthetic and versions are hard-coded SHA1s?

Comment (fviernau) [reply analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml:23]: Does the synthetic test project really specify the dependency on `GoDep::golang.org/x/sys` explicitly (along with a SHA1)?

Comment (sschuberth) [reply analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml:23]: No, it's an implicitly dependency that's somehow introduced. But I still wouldn't have expected it to change unless maybe the Go compiler version changes.

Comment (mnonnenmacher) [reply analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml:23]: This is a transitive dependency. We should monitor if this happens more often and if so replace the direct dependency with another one that does not depend on a changing package.

Comment (sschuberth) [reply analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml:23]: A transitive dependency of what, of `go-isatty`? If so, it should be nested under `go-isatty`, no?

Comment (mnonnenmacher) [reply analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml:23]: GoDep does not support dependency trees, it's always a flat list.

Comment (sschuberth) [reply analyzer/src/funTest/assets/projects/synthetic/godeps-expected-output.yml:23]: Oh, I wasn't aware of that. But wouldn't it be the job of our GoDep package manager implementation to still *create* a tree?

**Pull request merged**